### ISSUE TITLE
Hosting redundant files

### ISSUE THREAD

**Issue by terfilip** [qmcs/qmcs.github.io#52]
Is there a reason to host .psd files and bootstrap?
- The psd files take up a lot of space and slow down uploads of a new branch
- Bootstrap isn't too big but unless there's a reason to keep it here, we should switch to the CDN

If you want to host the psd files for future development, I'll instead make a `site-resources` or something repo so it doesn't get carried around with the site.

**Comment by dimazest** [comment]
Yeah, we should have a separate repo for graphic sources and similar things.

I'm not sure whether bootstrap is a problem. Our build script can pull bootstrap from somewhere, so we don't need to put it to git.

**Comment by terfilip** [comment]
I'll make the resources repo, guess we'll keep bootstrap around

**Comment by terfilip** [comment]
Ok done, moved the psd files to [site-resources](https://github.com/qmcs/site-resources), any other files there that are useful for site dev can be uploaded there.

cc @HSkogmo
